app/views/layouts/application.html.erb

```diff
@@ -15,6 +17,10 @@
 
   <body>
     <%= render "shared/header" %>
-    <%= yield %>
-  </body>
+    <div id="su-wrap"><%# required to push the footer to the bottom of the page, rather than the viewport %>
+      <div id="su-content">
+        <%= yield %>
+      </div>
+    </div>
```

Comment:
Also, I found that the su-wrap/su-content divs are also no longer needed: https://github.com/sul-dlss/argo/pull/4150/files#diff-a81956f33beaf83f584f6bc14d1fa1fa4725d1f8a8954412d371d55a7c4746bb